src/engine/test/assets/decoders/syscollector_process_save.yml

```diff
@@ -0,0 +1,162 @@
+
+  - check:
+      - _syscollectorEventOriginal.process.size: +not_exists
+    map:
+      _syscollectorEventOriginal.process.size: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.vm_size: +not_exists
+    map:
+      _syscollectorEventOriginal.process.vm_size: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.resident: +not_exists
+    map:
+      _syscollectorEventOriginal.process.resident: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.share: +not_exists
+    map:
+      _syscollectorEventOriginal.process.share: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.start_time: +not_exists
+    map:
+      _syscollectorEventOriginal.process.start_time: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.pgrp: +not_exists
+    map:
+      _syscollectorEventOriginal.process.pgrp: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.session: +not_exists
+    map:
+      _syscollectorEventOriginal.process.session: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.nlwp: +not_exists
+    map:
+      _syscollectorEventOriginal.process.nlwp: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.tgid: +not_exists
+    map:
+      _syscollectorEventOriginal.process.tgid: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.tty: +not_exists
+    map:
+      _syscollectorEventOriginal.process.tty: "NULL"
+
+  - check:
+      - _syscollectorEventOriginal.process.processor: +not_exists
+    map:
+      _syscollectorEventOriginal.process.processor: "NULL"
+
+  - map:
+      _query: +s_concat/agent /$_syscollectorEventOriginal.agent.id/ process save /$_syscollectorEventOriginal.ID /|/$_syscollectorEventOriginal.process.timestamp/|/$_syscollectorEventOriginal.process.pid/|/$_syscollectorEventOriginal.process.name/|/$_syscollectorEventOriginal.process.state/|/$_syscollectorEventOriginal.process.ppid/|/$_syscollectorEventOriginal.process.utime/|/$_syscollectorEventOriginal.process.stime/|/$_syscollectorEventOriginal.process.cmd/|/$_argvs_csv_str/|/$_syscollectorEventOriginal.process.euser/|/$_syscollectorEventOriginal.process.ruser/|/$_syscollectorEventOriginal.process.suser/|/$_syscollectorEventOriginal.process.egroup/|/$_syscollectorEventOriginal.process.rgroup/|/$_syscollectorEventOriginal.process.sgroup/|/$_syscollectorEventOriginal.process.fgroup/|/$_syscollectorEventOriginal.process.priority/|/$_syscollectorEventOriginal.process.nice/|/$_syscollectorEventOriginal.process.size/|/$_syscollectorEventOriginal.process.vm_size/|/$_syscollectorEventOriginal.process.resident/|/$_syscollectorEventOriginal.process.share/|/$_syscollectorEventOriginal.process.start_time/|/$_syscollectorEventOriginal.process.pgrp/|/$_syscollectorEventOriginal.process.session/|/$_syscollectorEventOriginal.process.nlwp/|/$_syscollectorEventOriginal.process.tgid/|/$_syscollectorEventOriginal.process.tty/|/$_syscollectorEventOriginal.process.processor
```

Comment:
Please, check this line, as the Github interpreter displays it in a strange color.

Reply:
I couldn't find what that different color could mean, I'm just guessing that it could be a consequence of the line length.